usagov/vote-gov-drupal
# Vote 983 -> Translations Url Test

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-983](https://bixal-projects.atlassian.net/browse/VOTE-983)

## Description (optional)
This PR will add test coverage to verify that each language option has the correct href path to direct the user to the right URL once it is clicked. 

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. start local 
3. cd into `testing`
4. run `npm run cy:open`
5. select the `translations-urls.cy.js` file and verify test passes
